### DISCUSSION TITLE
How can I get the model for test in "Filament Materials Guide"

### DISCUSSION THREAD

**Discussion by zhenhuafan** [google/filament#6706] [Q&A] [answered]
I want to test the materials like below:

 
![cloth](https://user-images.githubusercontent.com/78893358/229711437-e7daf05b-595e-46f1-890f-6aa273a34403.JPG)

![head](https://user-images.githubusercontent.com/78893358/229711506-fbd6af20-f53e-42ef-b224-e356f448b956.JPG)

Can I get the models's mesh and normal_map，etc. ?

**Comment by bejado** [comment, marked as answer]
The cloth model is in our repo: https://github.com/google/filament/tree/main/assets/models/cloth
The head model is available on SketchFab: https://sketchfab.com/3d-models/head-lee-perry-smith-4d07eb2030db4406bc7eee971d1d3a97
